packages/subgraph/src/mappings/superToken.ts

```diff
@@ -452,6 +453,17 @@ function _createTransferEventEntity(event: Transfer): void {
 }
 
 export function handleApproval(event: Approval): void {
+    const eventId = createEventID("Approval", event);
+    const ev = new ApprovalEvent(eventId);
+    initializeEventEntity(ev, event, []);
```

Comment:
relevant addresses need to go here (token, owner, to)

same goes for the other approval event

Reply:
Didn't change yet? Should do a quick search of `initializeEventEntity(ev, event, []);` over the code base to quickly find the spots where adding the related addresses might be missing.

Reply:
yeah good point